meta/main.yml

```diff
@@ -3,24 +3,16 @@
 dependencies: []
 
 galaxy_info:
-  role_name: '_template'
+  role_name: 'php_fpm'
   author: 'Adfinis SyGroup AG'
-  description: ''
+  description: 'This role is used to install and configure php-fpm'
   company: 'Adfinis SyGroup AG'
   license: 'GNU General Public License v3'
   min_ansible_version: '2.0.0'
   platforms:
-    - name: Debian
-      versions:
-        - wheezy
-        - jessie
-        - stretch
-    - name: Ubuntu
-      versions:
-        - trusty
-        - xenial
     - name: EL
       versions:
-        - 6
         - 7
-  galaxy_tags: []
+  galaxy_tags:
```

Comment:
you could add `php7` and `centos`/`rhel` to tags, as this role will only work on rh-systems